modules/apps/object/object-web/src/main/resources/META-INF/resources/js/tests/ObjectField/TranslationOptionsContainer.spec.tsx

```diff
@@ -0,0 +1,173 @@
+	).toBeNull();
+};
+
+const renderComponent = (props = {}) => {
+	render(<TranslationOptionsContainer {...defaultProps} {...props} />);
+};
+
+describe('When the feature flag LPD-32050 is enabled', () => {
+	beforeAll(() => {
+		global.Liferay = {
+			...global.Liferay,
+			FeatureFlags: {
+				...global.Liferay?.FeatureFlags,
+				'LPD-32050': true,
+			},
+		};
+	});
+
+	test.each`
+		businessType             | system   | shouldRenderAssertion
+		${'Aggregation'}         | ${false} | ${'render'}
+		${'Attachment'}          | ${false} | ${'not render'}
+		${'AutoIncrement'}       | ${false} | ${'render'}
+		${'Boolean'}             | ${false} | ${'not render'}
+		${'Date'}                | ${false} | ${'not render'}
+		${'DateTime'}            | ${false} | ${'not render'}
+		${'Decimal'}             | ${false} | ${'not render'}
+		${'Encrypted'}           | ${false} | ${'render'}
+		${'Formula'}             | ${false} | ${'render'}
+		${'Integer'}             | ${false} | ${'not render'}
+		${'LongInteger'}         | ${false} | ${'not render'}
+		${'LongText'}            | ${false} | ${'not render'}
+		${'MultiselectPicklist'} | ${false} | ${'not render'}
+		${'RichText'}            | ${false} | ${'not render'}
+		${'Picklist'}            | ${false} | ${'not render'}
+		${'PrecisionDecimal'}    | ${false} | ${'not render'}
+		${'Text'}                | ${false} | ${'not render'}
+		${'Aggregation'}         | ${true}  | ${'render'}
+		${'Attachment'}          | ${true}  | ${'render'}
+		${'AutoIncrement'}       | ${true}  | ${'render'}
+		${'Boolean'}             | ${true}  | ${'render'}
+		${'Date'}                | ${true}  | ${'render'}
+		${'DateTime'}            | ${true}  | ${'render'}
+		${'Decimal'}             | ${true}  | ${'render'}
+		${'Encrypted'}           | ${true}  | ${'render'}
+		${'Formula'}             | ${true}  | ${'render'}
+		${'Integer'}             | ${true}  | ${'render'}
+		${'LongInteger'}         | ${true}  | ${'render'}
+		${'LongText'}            | ${true}  | ${'render'}
+		${'MultiselectPicklist'} | ${true}  | ${'render'}
+		${'RichText'}            | ${true}  | ${'render'}
+		${'Picklist'}            | ${true}  | ${'render'}
+		${'PrecisionDecimal'}    | ${true}  | ${'render'}
+		${'Text'}                | ${true}  | ${'render'}
+	`(
+		'the object field $businessType system $system should $shouldRenderAssertion the translation alert',
+		({businessType, shouldRenderAssertion, system}) => {
+			renderComponent({values: {businessType, system}});
+
+			if (shouldRenderAssertion === 'render') {
+				alertIsRendered();
+			}
+			else {
+				alertNotRendered();
+			}
+		}
+	);
+});
+
+describe('When the feature flag LPD-32050 is disabled', () => {
+	beforeAll(() => {
+		global.Liferay = {
+			...global.Liferay,
+			FeatureFlags: {
+				...global.Liferay?.FeatureFlags,
+				'LPD-32050': false,
+			},
+		};
+	});
+
+	test.each`
+		businessType             | system   | shouldRenderAssertion
```

Comment:
what do you think to rename `shouldRenderAssertion` to `shouldRenderAlert`?

Reply:
Hi @rebsilva, see my changes [here](https://github.com/liferay-bpm/liferay-portal/compare/748097b8e78de1b070f450e90ee195ae8390c28d..fdc158fe2ed7f9a6d5b8e4c7bd8f4f23c42b5b35)